src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -1073,8 +1080,8 @@ private enum OptionFlags : uint
             HasHeader = 0x02,
             AllowQuoting = 0x04,
             AllowSparse = 0x08,
-
-            All = TrimWhitespace | HasHeader | AllowQuoting | AllowSparse
+            ReadMultilines = 0x10,
+            All = TrimWhitespace | HasHeader | AllowQuoting | AllowSparse | ReadMultilines
```

Comment:
After refactoring `_readMultilines ` from a private member into the `OptionFlags ` enum, I realized there's no need to change the `.Save()` logic to serialize the loader, because it simply writes 2 bytes representing the `OptionFlags _flags` of the Loader. Because of this, no need to change the Save/Load logic, and also no need to add a backward compatibility test.